packages/block-editor/src/components/block-card/index.js

```diff
@@ -17,6 +22,7 @@ function BlockCard( {
 	blockType,
 	parentBlockClientId,
 	handleBackButton,
+	isSynced,
```

Comment:
I think it would make better to add `className` here instead of this specific prop. It would save us possible deprecations in the future. What do you think?

Reply:
Good idea - I will take a look at that tomorrow.